python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/vlans.schema.yml

```
@@ -43,6 +43,15 @@ keys:
           items:
             type: str
             description: Trunk Group Name.
+        e_tree:
+          type: dict
+          keys:
+            leaf_role:
+              type: bool
+              description: Set the VLAN into the E-Tree leaf role. By default all VLANs are in root role
+            remote_leaf_host_drop:
+              type: bool
+              description: Enables remote leaf hosts to instead be installed as explicit drop routes in the local FDB
```

```suggestion
              description: Enables remote leaf hosts to instead be installed as explicit drop routes in the local FDB.
```
It also requires to define the leaf_role as true. So update the description.

---

Done